keyboards/ut47/keymaps/nordic/keymap.c

```diff
@@ -0,0 +1,178 @@
+/* Copyright 2018 Carlos Filoteo
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#ifdef LED_ENABLE
+  #include "protocol/serial.h"
+
+#endif
+
+#include "keymap_norwegian.h"
+#include "keymap.h"
```

Comment:
This include isn't needed.  Just the `keymap_norwegian.h` is, since that includes it.

Reply:
Oops, forgot that one. Fixing it now!